pandas/io/tests/test_stata.py

```diff
@@ -780,8 +780,14 @@ def test_drop_column(self):
         expected = expected[columns]
         dropped = read_stata(self.dta15_117, convert_dates=True,
                              columns=columns)
-
         tm.assert_frame_equal(expected, dropped)
+
+        columns = ['int_', 'long_', 'byte_']
+        expected = expected[columns]
+        reordered = read_stata(self.dta15_117, convert_dates=True,
+                               columns=columns)
```

Comment:
There are tests on invalid columns in this test function already just a bit
below this line. I just added the case of columns with different order.
On Aug 4, 2015 12:06 AM, "Jeff Reback" <notifications@github.com> wrote:

> In pandas/io/tests/test_stata.py
> <https://github.com/pydata/pandas/pull/10739#discussion_r36137343>:
>
> >          tm.assert_frame_equal(expected, dropped)
> > +
> > +        columns = ['int_', 'long_', 'byte_']
> > +        expected = expected[columns]
> > +        reordered = read_stata(self.dta15_117, convert_dates=True,
> > +                               columns=columns)
>
> can you tests with an invalid column? (not exactly sure what we should do
> though), could reindex, KeyError or ignore.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pydata/pandas/pull/10739/files#r36137343>.
>